fonts.css

```diff
@@ -0,0 +1,7 @@
+@font-face{
+	font-family: 'Montserrat';
+	src: url('./fonts/Montserrat-Regular.otf') format('otf'),
+	url('./fonts/Montserrat-Regular.ttf') format('ttf');
+	font-weight: normal;
+	font-style: normal;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/tesla_car/pullRequest?prid=5929182)